pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go

```diff
@@ -1159,7 +1159,7 @@ func TestPipelineRunSpec_Invalidate(t *testing.T) {
 		),
 		withContext: cfgtesting.EnableAlphaAPIFields,
 	}, {
-		name: "computeResources disallowed without alpha feature gate",
+		name: "computeResources disallowed without beta feature gate",
```

Comment:
I think we need a case where the flag is set to beta and the validation is valid, under `TestPipelineRunSpec_Validate`

Reply:
OK. Sure.